mailroom.py

```diff
@@ -0,0 +1,275 @@
+#   $Id: play01.py,v 1.11 2017/02/12 20:27:25 larry Exp $
+
+# Various import
+# ---------------------------------------------------------------
+import sys
+import pickle
+import os.path
+import datetime
+# ---------------------------------------------------------------
+
+# Create dictionary
+donors = {
+"Bugs Bunny":[1000, 2000, 3000],
+"Elmer Fudd":[500, 1200, 1600],
+"Foghorn Leghorn":[800, 1600, 2400],
+"Daffy Duck":[550, 750, 2400],
+"Johnny Quest":[300, 600, 900],
+"Marvin Martian":[5000, 10000, 20000],
+"Porky Pig":[2500, 2800, 3000]}
+
+# Open the pickle
+try:
+    # Open the pickle
+    file_obj = open("/home/larry/UW/Play/donors", 'wb')
```

Comment:
When working with file paths, python has some functions that are super nice so that things can be run without hardcoding file paths.

In this instance you can do:
```
import os

file_obj = open(os.path.abspath('donors')), 'wb')
```